docs/maui/converters/enum-to-bool-converter.md

```diff
@@ -0,0 +1,172 @@
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             xmlns:mylittleapp="clr-namespace:MyLittleApp"
+             x:Class="MyLittleApp.MainPage">
+    <ContentPage.Resources>
+         <ResourceDictionary>
+             <toolkit:EnumToBoolConverter x:Key="MobileConverter">
+                <toolkit:EnumToBoolConverter.TrueValues>
+                    <mylittleapp:MyDevicePlatform>Android</mylittleapp:MyDevicePlatform>
+                    <mylittleapp:MyDevicePlatform>iOS</mylittleapp:MyDevicePlatform>
+                </mct:EnumToBoolConverter.TrueValues>
+            </mct:EnumToBoolConverter>
+         </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <VerticalStackLayout>
+        <Picker ItemsSource="{Binding Platforms}"
+                SelectedItem="{Binding SelectedPlatform}" />
+        <Label IsVisible="{Binding SelectedPlatform, Converter={StaticResource MobileConverter}}"
+               Text="I am visible when the Picker value is Android or iOS."/>
+    </VerticalStackLayout>
+</ContentPage>
+```
+
+It is also possible to pass the converter parameter:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="MyLittleApp.MainPage">
+    <ContentPage.Resources>
+         <ResourceDictionary>
+             <toolkit:EnumToBoolConverter x:Key="PlatformConverter" />
+         </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <VerticalStackLayout>
+        <Picker ItemsSource="{Binding Platforms}"
+                SelectedItem="{Binding SelectedPlatform}" />
+        <Label IsVisible="{Binding SelectedPlatform, Converter={StaticResource PlatformConverter}, ConverterParameter={x:Static vm:MyDevicePlatform.Tizen}}"
+               Text="I am visible when the Picker value is Tizen."/>
+    </VerticalStackLayout>
+</ContentPage>
+```
+
+### C#
+
+The `EnumToBoolConverter` can be used as follows in C#:
+
+```csharp
+class EnumToBoolConverterPage : ContentPage
+{
+    public EnumToBoolConverterPage()
+    {
+        var picker = new Picker();
+        picker.SetBinding(Picker.ItemsSourceProperty, nameof(ViewModel.Platforms));
+        picker.SetBinding(Picker.SelectedItemProperty, nameof(ViewModel.SelectedPlatform));
+
+        var label = new Label
+        {
+            Text = "I am visible when the Picker value is Tizen."
+        };
+
+		label.SetBinding(
+			Label.IsVisibleProperty,
+			new Binding(
+				nameof(ViewModel.SelectedPlatform),
+				converter: new EnumToBoolConverter(),
+                converterParameter: MyDevicePlatform.Tizen));
+
+		Content = new VerticalStackLayout
+        {
+            Children = { picker, label }
+        };
```

Comment:
@jfversluis are you able to check if this renders correctly in the preview please?

Reply:
Looks good to me!